scss/atoms/modals/_modals.scss

```diff
@@ -0,0 +1,33 @@
+/*
+* ==========================================================================
+* Modals
+* ==========================================================================
+*/
+
+@import
+	"dist/chassis",
+	"functions",
+	"mixins";
+
+.modal {
+	@include modal();
+
+	.modal-header {
+		@include modal-header();
+		h4 {
```

Comment:
The default one as styled in typography.scss has wide top and bottom margins so I had to override that.